app/helpers/test_passages_helper.rb

```diff
@@ -0,0 +1,15 @@
+module TestPassagesHelper
+  PASS_TEST_PERCENT = 0.85
+
+  def test_result(test_passage)
+    if percentage_of_correct_answers >= PASS_TEST_PERCENT
```

Comment:
```suggestion
    percentage_of_correct_answers >= PASS_TEST_PERCENT
```

Условное выражение избыточно.

Reply:
Есть ожидания что метод `#test_result` вернет `success` или `failure`, поэтому лучше назвать `success?` и сделать это методом модели.